app/src/main/res/layout/activity_event_picking.xml

```diff
@@ -1,33 +1,56 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".ui.eventSelector.EventPickingActivity">
+                                             xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
That's not how the other XML files were indented. I don't mind this one, but we have to stay coherent and use the same indentation scheme everywhere!